### DISCUSSION TITLE
Model does not trade.

### DISCUSSION THREAD

**Discussion by farzinaa** [AI4Finance-Foundation/FinRL#869] [Q&A]
I have issues training for larger data sets as the test result does not change (model does not trade during testing). This issue is fixed when the data  set is specific to certain time periods for example for 
start_date = '2021-01-01'
end_date = '2022-05-01'
start_test = '2022-05-02'
end_test = '2023-06-01'
for BTC USDT or ETH USDT. 

I imagine this has to do with the answer mentioned in [issue #465](https://github.com/AI4Finance-Foundation/FinRL/issues/465#issuecomment-1046327975), namely 

>  Also, the training data distribution is different to test data distribution, like the training model on Tesla before 2020, and testing it afterward may lead to sub-optimal results.

Any suggestions how one can manage this? Is an Ensemble strategy the answer?

**Comment by YangletLiu** [comment]
Is it solved now?

**Reply by farzinaa** [reply]
YES. This was a mistake on my part as it turns out that my turbulence array was returning 0s'! 
I apologize for creating an issue for a non issue.

**Reply by YangletLiu** [reply]
No problem.  
Our community is "big", and I am not able to work closely with you.
Would be great if you can help other members. Thanks!
